x-pack/test/functional/apps/dashboard/group3/reporting/screenshots.ts

```diff
@@ -172,10 +172,12 @@ export default function ({
         await PageObjects.dashboard.loadSavedDashboard('Ecom Dashboard');
         await PageObjects.reporting.openExportTab();
         await PageObjects.reporting.clickGenerateReportButton();
-        await PageObjects.share.closeShareModal();
 
         const url = await PageObjects.reporting.getReportURL(60000);
         const res = await PageObjects.reporting.getResponse(url ?? '');
+        if (await PageObjects.share.isShareMenuOpen()) {
+          await PageObjects.share.closeShareModal();
```

Comment:
Wrapping `closeShareModal` actually doesn't make a substantial difference, since it's definition already performs this check, see https://github.com/elastic/kibana/blob/main/test/functional/page_objects/share_page.ts#L20-L26